roslibrust_codegen_macro/src/lib.rs

```diff
@@ -46,9 +46,9 @@ pub fn find_and_generate_ros_messages_relative_to_manifest_dir(
 
     std::env::set_current_dir(env!("CARGO_MANIFEST_DIR")).expect("Failed to set working dir");
     for path in &mut paths {
-        *path = path
-            .canonicalize()
-            .expect(&format!("Failed to canonicalize path: {path:?}"));
+        *path = path.canonicalize().unwrap_or_else(|err| {
```

Comment:
This would create an error message if the resulting path didn't exist, but it didn't say why, now it does.